src/js/homebase.js

```diff
@@ -0,0 +1,66 @@
+let homebase = {};
+
+homebase.state = {
+	"size": [ 4, 4 ], // How wide, how tall?
+	"1P": {
+		"coords": [	0, 0 ], // Where is 1P in the base? -- X, Y coords
+		"orientation": "N", // What direction is 1P facing?
+		"status": "A" // What color is 1P?
+	}
+};
+
+function updateHomeBaseView(coords = [ 0, 0 ]) {
+  let homeBase = document.getElementById("js-home-base"),
+      player1 = document.getElementById("js-player1");
+  
+  homeBase.querySelectorAll("div.home-base__square")[get1dCoordsFrom2d(coords)].appendChild(player1);
+}
+
+document.addEventListener("keydown", (event) => {
+  const directionKeys = {
+        "W": "north",
+        "D": "east",
+        "S": "south",
+        "A": "west",
+        "ARROWUP": "north",
+        "ARROWRIGHT": "east",
+        "ARROWDOWN": "south",
+        "ARROWLEFT": "west",
+    },
+        direction = directionKeys[event.key.toUpperCase()];
+
+  if (typeof direction !== "undefined") {
```

Comment:
I wonder if this'd be more clearly expressed some other way, like how we can use `Array.contains()`, I wonder if there's a similar method on the `Object` prototype